config/config.go

```diff
@@ -128,6 +129,46 @@ func (cfg *AuctionTimeouts) validate(errs configErrors) configErrors {
 	return errs
 }
 
+func (data *ExternalCache) validate(errs configErrors) configErrors {
+	if data.Host == "" {
+		if data.Path != "" {
+			// Throw an error because we have a blank Host nad a non-blank Path
+			errs = append(errs, fmt.Errorf("Could not parse URL with empty Host: %s \n", data.Host+data.Path))
+		}
+		// Blank host and path are valid and will not throw an error
+		return errs
+	}
+
+	// Add "/" suffix to the path if missing
```

Comment:
Nitpick: Please update the comment. This is actually ensuring a prefix, not a suffix.

Reply:
I should really review this comments because they are definitely poorly written. Changed